pandas/core/generic.py

```diff
@@ -4964,6 +4964,21 @@ def sample(
         dog          4          0                  2
         fish         0          0                  8
 
+
+        An upsample sample of the ``DataFrame`` with replacement:
+        Note that `replace` parameter has to be `True` for `frac` parameter > 1.
```

Comment:
do we have this in the Notes section as well (about replace=True for frac > 1)

Reply:
I added the Notes session